readme.md

```diff
@@ -19,7 +19,9 @@ The dashboard in the example has three items:
 
```

Comment:
Can you update lines 16-18 above?
Please remove the punctuation for these list items as follows:

- The "Products" grid
- The "Sales per Product" chart
- The "Orders" grid